main.go

```diff
@@ -39,6 +39,11 @@ func main() {
 			Usage:  "Kubernetes helm release",
 			EnvVar: "PLUGIN_RELEASE,RELEASE",
 		},
+		cli.StringSliceFlag{
+			Name:   "helm_repos",
+			Usage:  "Repos helm should add",
+			EnvVar: "PLUGIN_HELM_REPOS,HELM_REPOS",
+		},
```

Comment:
My initial assumption was that multiple repos could be used at the same time (dependencies) - but `helm dep up` packages dependencies with parent charts and thus I can't think of any other reason for accepting a list of repositories. 

If we only want to take in a single repository, I believe we shouldn't use the `name=url` format and just accept repo `url` - the  chart should then be referenced as `repo/my-chart`